src/documentation/ToolkitDocumentation/ToolkitContent.tsx

```diff
@@ -38,6 +38,27 @@ export const imageUrlBuilder = unconfiguredImageUrlBuilder()
   .dpr(window.devicePixelRatio ?? 1)
   .quality(defaultQuality);
 
+const getImageDimensions = (imageRef: string): number[] | undefined => {
```

Comment:
This will work fine. We also have the option of pulling more information from Sanity which provides some of this in a more convenient form (I actually hadn't realised we needed to add to the query for this, in the Gatsby setup I more often use it's done differently). In that case we'd need to add to the query to resolve the reference to images in each content section and retrieve dimensions.aspectRatio.

Reply:
I think it's worth adding a test for an image in ToolkitContent.test.tsx. You can use the same style of [jest snapshot testing](https://jestjs.io/docs/snapshot-testing).